endpointmanager/pkg/endpointmanager/fhirendpoint.go

```diff
@@ -99,3 +104,16 @@ func NormalizeWellKnownURL(url string) string {
 	}
 	return normalized
 }
+
+// Prepends url with https:// and appends with $versions if needed
+func NormalizeVersionsURL(url string) string {
```

Comment:
Why are all of these normalize functions here when they're only used in the capabilityquerier?

Reply:
Good question -- I think its because these were intended to be general utility functions which do belong in the endpointmanager but it turns out the we've only used them (so far) in the CapabilityQuerier. I think if this was the _only_ dependence capabilityQuerier had on endpointmananger then we would want to remove it

Reply:
okay, that makes sense to me!